POEApi.Model/JSONProxy/Stash.cs

```diff
@@ -26,6 +26,14 @@ public class Requirement
         public int DisplayMode { get; set; }
     }
 
+    public class IncubatedDetails
+    {
+        public string Name { get; set; }
+        public int Level { get; set; }
```

Comment:
Is `MinimumLevel` a more accurate name for all uses of the variable?

Reply:
As above